build.gradle

```diff
@@ -782,7 +606,7 @@ task mapNamedJar(type: MapJarTask, dependsOn: ["mergeV2", "unpickIntermediaryJar
 	output = namedJar
 	input = unpickedJar
 	mappings = mergeV2.output
-	classpath.from fileTree(libraries)
+	classpath.from configurations.minecraftLibraries
```

Comment:
Hmm, did you check that the file collection => file tree conversion in MapJarTask works properly for collections with multiple root directories? (I think it should, but double-checking just in case)